### PR TITLE
[W4.7][W11-A2]Kevin Leonardo

### PR DESCRIPTION
**Pull request by k-l-a**
Pull Request for LO Encapsulation.
Change the visibility of variable member feedbackToUser from CommandResult class for
protection. Variable was changed from public to private.
New method, getFeedbackMessage, was implemented to access feedbackToUser variable from
outside.
Usage of the variable was appropriately updated to preserve functionality.

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -33,4 +33,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * Returns the feedback message to be shown to user
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/CommandResult.java]: :+1:

File: test/java/seedu/addressbook/commands/AddCommandTest.java [hunk 2]
```diff
@@ -139,7 +139,7 @@ public void addCommand_addressBookAlreadyContainsPerson_addressBookUnmodified()
         CommandResult result = command.execute();
 
         assertFalse(result.getRelevantPersons().isPresent());
-        assertEquals(AddCommand.MESSAGE_DUPLICATE_PERSON, result.feedbackToUser);
+        assertEquals(AddCommand.MESSAGE_DUPLICATE_PERSON, result.getFeedbackMessage());
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/AddCommandTest.java]: :100: good job on updating the tests as well!

**Comment by okkhoy** [comment]
@k-l-a 
Some comments added. Please ack & close the PR after reading comments.

**Comment by k-l-a** [comment]
Thank you for the review. Will now close the PR.

**Pull request closed without merging**